homeassistant/components/tibber/diagnostics.py

```diff
@@ -13,19 +13,23 @@ async def async_get_config_entry_diagnostics(
     hass: HomeAssistant, config_entry: ConfigEntry
 ) -> dict:
     """Return diagnostics for a config entry."""
-    tibber_connection: tibber.Tibber = hass.data[DOMAIN]
-
     diagnostics_data = {}
```

Comment:
There should be no need to use this temporary dictionary, instead, directly return the resulting dictionary at the end of the method.

Reply:
@frenck Not sure I follow you, sorry.
diagnostics_data will now include two keys, an "api_connection" and a "homes" where "homes" is a json structure.
I guess you suggest to return just the homes dictionary but then the "api_connection" information will be missing.

Reply:
Just to this in the end:
```py
return {
  "api_connection": ...,
  "homes": ...,
}
```